### DISCUSSION TITLE
dev-3 uses 100% CPU, dev-2 uses 50% CPU?

### DISCUSSION THREAD

**Discussion by Miros77** [MinicraftPlus/minicraft-plus-revived#656] [Q&A] [answered]
dev-2
![11](https://github.com/MinicraftPlus/minicraft-plus-revived/assets/59547044/b6bb7655-60c7-4503-b496-ba5b8f46dce1)
dev-3
![12](https://github.com/MinicraftPlus/minicraft-plus-revived/assets/59547044/290caf9d-e789-4709-8078-86f7d98a965e)

Intel(R) Pentium(R) CPU B960 @ 2.20GHz 2,20 GHz
6.00 GB (5.78 GB available)
windows 7 max is installed temporarily

**Comment by BenCheung0422** [comment]
This problem is being significantly addressed by #523 and #524.

**Comment by Litorom** [comment, marked as answer]
Closing via #523 and #524 merged
